nurseconnect/tasks.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import
+
+import json
+import requests
+from collections import Counter
+
+from celery.schedules import crontab
+from celery.task import periodic_task
+
+from django.contrib.auth.models import User
+
+from nurseconnect.services import get_clinic_code
+from nurseconnect.settings import JEMBI_URL, JEMBI_USERNAME, JEMBI_PASSWORD
+
+
+class JembiMetricsPoster(object):
+    """
+    Send metrics to Jembi
+    """
+    def send_metric(self, data):
+        headers = {"Content-type": "application/json"}
+        requests.post(
+            url=JEMBI_URL,
+            headers=headers,
+            data=json.dumps(data),
+            auth=(JEMBI_USERNAME, JEMBI_PASSWORD),
+            verify=False
+        )
+
+
+def nurses_registered():
+    """Returns the number of nurses registered on the system."""
+    num_nurses = User.objects.filter(is_staff=False).count()
+    data = {
+        "dataValues": [
+            {
+                "dataElement": "CSv1k6HyWaX",
+                "period": "201601",
+                "orgUnit": "Fws0A9spb9F",
+                "value": str(num_nurses)
```

Comment:
@smn perhaps it could be better to have saved as `constants` elsewhere.
I'll modify